cmd/kube-controller-manager/app/controllermanager.go

```diff
@@ -328,6 +332,7 @@ type ControllerContext struct {
 	ResyncPeriod func() time.Duration
 }
 
+// IsControllerEnabled check if the context's controllers enabled or not
```

Comment:
Small nits. Others lgtm.
```suggestion
// IsControllerEnabled checks if the context's controllers enabled or not
```